protocol/proofs/fdg-refund-mode.md

```diff
@@ -0,0 +1,95 @@
+# FaultDisputeGame Refund Mode
+
+## Context
+
+Dealing with `FaultDisputeGame` bonds appropriately is an important part of the current OP Stack
+incident response process. `FaultDisputeGame` stores player bonds inside of the `DelayedWETH`
+contract which imposes a 7 day delay on bond withdrawals. During this delay period, the 2/2 Safe
+managed jointly by the Security Council and the Optimism Foundation can hold bonds from specific
+games or from all games at the same time. Held bonds are transferred to this 2/2 Safe where they
+can be manually disbursed to the correct recipients.
+
+## Problem Statement
+
+Although `DelayedWETH` is an effective tool at preventing a buggy or malicious `FaultDisputeGame`
+from distributing bonds to a potential attacker, the actual process of returning those bonds back
+to the correct players is underspecified. Underspecified proceses can introduce significant stress
+to already stressful situations where executive decisions around fallbacks must be made quickly and
+confidently. We therefore want to introduce alternative low-stress options for bond recovery.
+
+## Proposed Solution
+
+We propose modifying the `FaultDisputeGame` to enter a "refund mode" if the game is blacklisted.
+When the `FaultDisputeGame` is in this refund mode it will distribute bonds back to the users who
+originally deposited those bonds.
```

Comment:
The major downside of this is that it makes attacks which were previously extremely expensive because you lose your bonds very cheap because everyone gets a refund of bonds. That's a very unfortunate change in incentives which makes griefing much more likely.

Reply:
Ah so this is an *optional* mechanism that acts as a default. `DelayedWETH` can still be used to remove bonds just like before, we're just optionally adding this extra mechanism that can be used in the case where removing bonds wouldn't be worth the effort.

Reply:
Ah, it just sounds like when we blacklist a game it will go into refund mode immediately which we wouldn't want. If would could build a refund script offline without it being too complex I'd be tempted to do that, but it may be easier to have it built into the contracts.

An offline script just gives us more flexibility - for example we might want to create a list of honest and dishonest actor addresses and run a script to pay out bonds appropriately or only refund actors on the honest list etc. We should write those scripts ahead of time for anything we're likely to need though.

Reply:
We could modify this so that refund mode begins after the airgap window -- that would solve this

Reply:
Yeah if it only kicked in after the air gap if manual intervention hadn't happened before then that would make sense.